### PR TITLE
novacustom tests

### PR DESCRIPTION
**Pull request by johanes2115**

### CODE REVIEW

File: platform-configs/novacustom-ns70mu.robot [hunk 1]
```diff
@@ -49,6 +49,7 @@ ${WIFI_CARD}=                                       Intel(R) Wi-Fi 6 AX201 160MH
 ${WIFI_CARD_UBUNTU}=                                Intel Corporation Wi-Fi 6 AX201 (rev 20)
 ${USB_MODEL}=                                       USB Flash Memory
 ${EXTERNAL_HEADSET}=                                USB PnP Audio Device
+${WEBCAM_UBUNTU}=                                   Chicony Electronics Co., Ltd Chicony USB2.0 Camera
 
 ${DMIDECODE_SERIAL_NUMBER}=                         N/A
 ${DMIDECODE_FIRMWARE_VERSION}=                      Dasharo (coreboot+UEFI) v1.4.0
```
Comment (mkopec) [inline platform-configs/novacustom-ns70mu.robot:55]:
```suggestion
${DMIDECODE_FIRMWARE_VERSION}=                      Dasharo (coreboot+UEFI) v1.5.0
```

same for all other laptops

**Comment by mkopec** [comment]
@johanes2115  I see a lot of changes from https://github.com/Dasharo/open-source-firmware-validation/pull/16 - please change the target branch to that PR's branch to make this PR easier to review

**Review by mkopec — CHANGES_REQUESTED** [review]

**Comment by johanes2115** [comment]
> @johanes2115 I see a lot of changes from #16 - please change the target branch to that PR's branch to make this PR easier to review

@mkopec I changed the base branch.

File: regression-novacustom.sh [hunk 2]
```diff
@@ -41,55 +36,57 @@ handle_ctrl_c() {
...
+  # "wifi-bluetooth-support"
+  # "audio-subsystem"
+  # "nvme-support"
+  # "usb-camera"
+  # "sd-card-reader"
+  # "usb-type-c"
+  # "cpu-status"
+  # "ec-and-super-IO"
+  # "platform-suspend-and-resume"
+  # "firmware-bulding-locally"
```
Comment (miczyg1) [inline regression-novacustom.sh:56]: Why are these commented out?

File: variables.robot [hunk 3]
```diff
@@ -8,212 +8,53 @@ ${MEMTEST_HEADER}=          Memtest86+ 5.01 coreboot
...
-...                         platform=apu7    board-revision=a    env=prod
-...                         platform_vendor=PC Engines    firmware_type=BIOS
+&{RTE33}=                   ip=192.168.10.107
+...                         platform=msi-pro-z690-a-wifi-ddr4
+...                         sonoff_ip=192.168.10.21
+&{RTE34}=                   ip=192.168.10.199
+...                         platform=msi-pro-z690-a-wifi-ddr4
+...                         sonoff_ip=192.168.10.169
+&{RTE39}=
+...                         sonoff_ip=192.168.10.69
```
Comment (miczyg1) [inline variables.robot:24]: I don't understand why we leave just selected platforms. Especially that we leave only MSI DDR4 boards, but MSI DDR5 boards are removed

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: variables.robot (hunk 3 above)
Comment (macpijan) [reply variables.robot:24]: @miczyg1 We need to get rid of this whole CONFIG/RTE concept, as not all stands have RTE (like this one). It made it infeasible to run any tests on laptops. With changes in this MR, we were able to.

I removed unused platforms. Z790 could stay probably.

This branch is named `novacustom-WIP` for a reason. I plan that we clean this up after releases and propose some solution to the above problem.

Comment (miczyg1) [reply variables.robot:24]: Z690 DDR5 could stay too, I wish...

**Comment by macpijan** [comment]
Replaced with: https://github.com/Dasharo/open-source-firmware-validation/pull/125

**Pull request closed without merging**